op-program/host/config/config.go

```diff
@@ -317,7 +317,7 @@ func NewConfigFromCLI(log log.Logger, ctx *cli.Context) (*Config, error) {
 		l2ChainID = boot.CustomChainIDIndicator
 	} else if len(rollupCfgs) > 1 {
 		// L2ChainID is not applicable when multiple L2 sources are used and not using custom configs
-		l2ChainID = 0
+		l2ChainID = eth.ChainIDFromUInt64(0)
```

Comment:
assuming the default value is used on L100, this can be:
```suggestion
		l2ChainID = eth.ChainID{}
```

Reply:
Yep go spot, have fixed both.